include/universal/utility/evaluate_closure_of_number_systems.hpp

```diff
@@ -0,0 +1,202 @@
+#pragma once
+
+// Copyright (C) 2017 Stillwater Supercomputing, Inc.
+// SPDX-License-Identifier: MIT
+//
+// Author: Colby Wirth
+// Version: 18 April 2025
+// 
+// About:
+// This utility calls the buildClosurePlot function from genereateClosurePlots.hpp 
+// and generates the closure statistics for a templatized number system. 
+//
+//
+// ****************************************************************
+// *** Use 'processASystem()' function call to use this utility ***
+// ****************************************************************
+// 
+// This file is part of the universal numbers project, which is released under an MIT Open Source license.
+
+#include <iostream>
+#include <string>
+#include <string_view>
+#include <fstream>
+
+#include <universal/number/posit/posit.hpp>
```

Comment:
the posit/cfloat/lns includes should not be in this include file. 

the posit/cfloat/lns include files bring in those number systems and you are not using them 'specifically' in this file, so they should not be included. For include files, you should always strive to have a minimum set of includes that make the include file usable. In templated libs like universal, an include typically declares a set of parameterized classes and/or functions, that SHOULD be able to work with any of the types in the library. But you do not need to include them, as a matter of fact, you shouldn't include them. That is up to the calling application to bring together a context in which say a posit<> type is interpretable. But you need to leave them out of the defining include files like this.